### PR TITLE
Updates per review by Norbert Bensalem (ISO)

### PR DESCRIPTION
**Pull request by mbeckerle**
After discussion by DFDL WG 2023-06-01.

### CODE REVIEW

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# PAS Explanatory Report for Open Grid Forum (OGF) Data Format Description Language (DFDL)
+# PAS Explanatory Report for Open Grid Forum (OGF) Data Format Description Language (DFDL) Transposition
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:1]: Drop 'Transposition'

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 2]
```diff
@@ -29,8 +29,12 @@ This specification addresses a specific challenge faced in response to a need fo
 to be able to understand data regardless of source. 
 A language was needed capable of modeling a wide variety of existing text and binary data formats. 
 A working group was established at the Global Grid Forum (which later became the Open Grid Forum) 
-in 2003 to create a specification for such a language which has now become a Full Recommendation of the 
-OGF. 
+in 2003 to create a specification for such a language. 
+This specification progressed through the OGF process starting as a workgroup 'informational' draft, 
+then entering the OGF 2-stage process for "Recommendation Track" documents in 2011 by becoming an
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:34]: Duplication 'in 2011'

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 3]
```diff
@@ -312,6 +322,12 @@ dependent on these features accidentally.
 Experience reports about real use cases and the experience with the feature are required 
 before consideration of incorporation of an extension into a future version of the DFDL standard.   
 
+The OGF has already joined WG3 of SC38. 
+Preliminary contact has been established with WG5.
+Upon acceptance of this PAS submission by JTC 1 we expect to 
+increase the existing links with WG 5 and are committed to creating
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:328]: 'WG 5' -> 'WG5'

**Review by smhdfdl — CHANGES_REQUESTED** [review]

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md (hunk 1 above)
Comment (mbeckerle) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:1]: Transposition is the term for taking a PAS submission and flipping it into an ISO/IEC standard. 
Was added by ISO/IEC reviewer.

Comment (smhdfdl) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:1]: Ah ok

**Pull request merged**